students/kaczmarek_katarzyna/lesson_03_functions/fibonacci_numbers.py

```diff
@@ -0,0 +1,24 @@
+# Fibonacci: 1 1 2 3 5 8 13 ...
+def fib(n):
```

Comment:
0 traktujesz jako poprawna wartość czy nie?
Jeśli tak, to program będzie działać w nieskończoność, jeśli nie to nie ma zabezpieczeń na wartość 0 jako nie poprawna